Sources/StreamChatUI/ChatMessageList/ChatMessageListVC.swift

```diff
@@ -172,13 +172,15 @@ open class _ChatMessageListVC<ExtraData: ExtraDataTypes>:
     }
 
     open func attachmentViewInjectorClassForMessage(at indexPath: IndexPath) -> _AttachmentViewInjector<ExtraData>.Type? {
-        if messageForIndexPath(indexPath).imageAttachments.isEmpty == false {
+        let attachmentCounts = messageForIndexPath(indexPath).attachmentCounts
+
+        if attachmentCounts[.image, default: 0] > 0 {
             return components.galleryAttachmentInjector
-        } else if messageForIndexPath(indexPath).giphyAttachments.isEmpty == false {
+        } else if attachmentCounts[.giphy, default: 0] > 0 {
```

Comment:
hmm... can we just make sure only the non-empty keys are there? Then we could do `if attachmentCounts.keys.contains(.giphy)` which is a bit nicer.

Reply:
We can do it right away. Decided to check for the value instead of key existence to prevent errors if `attachmentCounts` start containing zeros one time

Reply:
That's what I meant by "can we just make sure only the non-empty keys are there" :) Since LLC creates the value we own the rules there.

Reply:
✅ Done. We already make sure we don't have `type: 0` counts [in here](https://github.com/GetStream/stream-chat-swift/blob/7d6b70519951cd71606066663a94667b004fb4bd/Sources/StreamChat/Database/DTOs/MessageDTO_Tests.swift#L473)